source/includes/examples/_find-the-center.html.md

```diff
@@ -0,0 +1,182 @@
+# Find the Center Example
+
+In this example, we'll walk you through the various statements that are part of the Find the Center project, including the simulator file and the Inkling file. Find the Middle was created as a very basic example of Inkling and how to connect to a simulator.
+
+The general premise of Find the Center is that the computer (via Inkling) takes a step (move) of -1, 0, or 1, and gets rewarded for staying in the center (1) for each round. The max is 2 and the min is 0.
+
+## Inkling File
+
+###### Schema
+
+```inkling
+schema GameState
+    Int32 value
+end
+```
+
+The `GameState` schema has one record, `value`, assigned the constrained type of Int32.
```

Comment:
I think you mean field not record.  A record is an "instance" of a schema (ie records have data in them).  A field is a description of a column of data within the record.  Int32 is unconstrained. I would instead say:
The `GameState` schema has one field, `value`, with a type of `Int32`. 

If you want to say more you could say: Using a type of `Int32` tells the BRAIN to be prepared to receive any number between -2,147,483,648 (minimum Int32) and 2,147,483,647 (maximum Int32) from the simulation.